include/Mutex.hpp

```diff
@@ -0,0 +1,26 @@
+/*
+** EPITECH PROJECT, 2024
+** Plaza
+** File description:
+** No file there , just an epitech header example .
+** You can even have multiple lines if you want !
+*/
+
+#pragma once
+#include <threads.h>
+#include <bits/pthreadtypes.h>
+#include <signal.h>
+#include <pthread.h>
+
+class Mutex {
+    public :
+        Mutex();
+        Mutex(const Mutex &mutex);
+        ~Mutex();
+        void lock();
+        void unlock();
+        void trylock();
+        Mutex &operator=(const Mutex &mutex);
+    private :
+        pthread_mutex_t _mutex;
+};
```

Comment:
Please add the missing trailing line
```suggestion
};

```

Reply:
Try to see if your IDE supports `.editorconfig` it should fix this issue in the future